src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -280,36 +292,57 @@ public async Task<string> CreatePullRequestAsync(string repoUri, PullRequest pul
                     SourceRefName = "refs/heads/" + pullRequest.HeadBranch,
                     TargetRefName = "refs/heads/" + pullRequest.BaseBranch
                 },
-                team,
-                repo);
+                projectName,
+                repoName);
 
             return createdPr.Url;
         }
 
+        /// <summary>
+        ///     Update a pull request with new information
+        /// </summary>
+        /// <param name="pullRequestUri">Uri of pull request to update</param>
+        /// <param name="pullRequest">Pull request info to update</param>
+        /// <returns></returns>
         public async Task UpdatePullRequestAsync(string pullRequestUri, PullRequest pullRequest)
         {
-            VssConnection connection = CreateConnection(pullRequestUri);
-            GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUri);
 
-            (string team, string repo, int id) = ParsePullRequestUri(pullRequestUri);
+            VssConnection connection = CreateVssConnection(accountName);
+            GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
 
             await client.UpdatePullRequestAsync(
                 new GitPullRequest
                 {
                     Title = pullRequest.Title,
                     Description = pullRequest.Description
                 },
-                team,
-                repo,
+                projectName,
+                repoName,
                 id);
         }
 
+        /// <summary>
+        ///     Merge a pull request
+        /// </summary>
+        /// <param name="pullRequestUrl">Uri of pull request to merge</param>
+        /// <param name="parameters">Settings for merge</param>
+        /// <returns></returns>
         public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters)
         {
-            VssConnection connection = CreateConnection(pullRequestUrl);
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUrl);
+
+            VssConnection connection = CreateVssConnection(accountName);
             GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
 
-            (string team, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
+            string commitToMerge = parameters.CommitToMerge;
+
+            // If the commit to merge is empty, look it up first.
+            if (string.IsNullOrEmpty(commitToMerge))
+            {
+                var prInfo = await client.GetPullRequestAsync(repoName, id);
```

Comment:
>var [](start = 16, length = 3)

I don't know what this var is. I think the general rule is "don't use var unless it's super explicit"

Reply:
yeah i just forgot to make it explicit.

Reply:
Happens to me all the time. I type "var" because it's easy, and usually autocorrect it to the right thing after the fact. I miss them all over the place.